src/model.rs

```diff
@@ -2789,4 +2808,67 @@ mod tests {
             assert_relative_eq!(stop_area.coord.lat, 0.0);
         }
     }
+
+    mod pickup_dropoff_harmonisation {
```

Comment:
You might be able to use the model builder to shorten the construction of a model for the test

Reply:
done in 2bc1c0438e4ad323e8829b42c39ef45f5e4c4a24

Reply:
Nice, much shorter! Thank you.